config/application.rb

```diff
@@ -10,8 +10,6 @@ module BandhistoryRails
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
-    # Enable templates for error pages rather than static HTML files
-    config.exceptions_app = self.routes
```

Comment:
Please restore at the bottom of this scope.

Reply:
done